### PR TITLE
Initial Animate Parser added to editor

### PR DESCRIPTION
**Pull request by AndrewJakubowicz**
Add initial support for parsing of Adobe Animate files from the thang editor.

### CODE REVIEW

File: app/views/editor/thang/ThangTypeEditView.coffee [hunk 1]
```diff
@@ -337,18 +341,43 @@ module.exports = class ThangTypeEditView extends RootView
...
+      noty({text: "Cancelled import of '.js' file", type:"info", timeout: 3000})
+      return
+    @reader = new FileReader()
+    @reader.onload = @onAnimateFileLoad
+    @reader.readAsText(file)
+
+  onAnimateFileLoad: (e) =>
+    result = @reader.result
+    parser = new AdobeAnimateParser(result)
+    parser.parse()
```
Comment (jmif) [inline app/views/editor/thang/ThangTypeEditView.coffee]: The parser will throw errors when things fail - are those handled / surfaced to user?

Comment (AndrewJakubowicz) [reply app/views/editor/thang/ThangTypeEditView.coffee]: Yeah. A noty was automatically made for me when I was testing.

**Comment by AndrewJakubowicz** [comment]
Going to update this to work with npm install.

File: app/views/editor/thang/ThangTypeEditView.coffee (hunk 1 above)
Comment (AndrewJakubowicz) [reply app/views/editor/thang/ThangTypeEditView.coffee]: Made this more explicit. Thanks

**Pull request merged**